docs/components/attention/elements.md

```diff
@@ -36,17 +36,17 @@
 ### Accessibility
 If the Attention element has "left" or "top" position, it should be placed before the target element in the DOM.
 
-Attention element handles accessibility automatically by wrapping its slotted content with a `div` with `role="tooltip"`, and setting an aria-describedby attribute on the target element.
+Attention element handles accessibility automatically by wrapping its slotted content with a `div` with either `role="tooltip"` for tooltip or `role="img"` for callout and popover, a default `aria-label` and setting an `aria-details` attribute on the target element. The default `aria-label` also supports i18n. 
```

Comment:
I got a bit lost in this sentence, especially around what `aria-label` refers to 😅  Maybe we could rephrase it a bit? here's an example:

...by wrapping its slotted content with a `div` that has a default `role` attribute (`role="tooltip"` for tooltips and `role="img"` otherwise), and a default localized `aria-label`. In addition, Attention automatically sets an `aria-details` on its target element, pointing to the slotted message element.